ITGlueAPI/Private/ApiKeys/Get-ITGlueAPIKey.ps1

```diff
@@ -0,0 +1,76 @@
+function Get-ITGlueAPIKey {
+<#
+    .SYNOPSIS
+        Gets the ITGlue API key
+
+    .DESCRIPTION
+        The Get-ITGlueAPIKey cmdlet gets the ITGlue API key from
+        the global variable and returns it as a SecureString
+
+    .PARAMETER PlainText
+        Decrypt and return the API key in plain text
+
+    .EXAMPLE
+        Get-ITGlueAPIKey
+
+        Gets the ITGlue API secret key global variable and returns an object
+        with the secret key as a SecureString
+
+    .EXAMPLE
+        Get-ITGlueAPIKey -PlainText
+
+        Gets the ITGlue API secret key global variable and returns an object
+        with the secret key as plain text
+
+    .NOTES
+        N/A
+
+    .LINK
+        https://itglue.github.io/ITGlue-PowerShellWrapper/site/Internal/Get-ITGlueAPIKey.html
+
+    .LINK
+        https://github.com/itglue/powershellwrapper
+
+#>
+
+    [CmdletBinding()]
+    Param (
+        [Parameter(Mandatory = $false)]
+        [switch]$PlainText
+    )
+
+    begin {}
+
+    process {
+
+        try {
+
+            if ($ITGlue_API_Key) {
+
+                if ($PlainText) {
+                    $Api_Key = [System.Runtime.InteropServices.Marshal]::SecureStringToBSTR($ITGlue_API_Key)
+
+                    ([System.Runtime.InteropServices.Marshal]::PtrToStringAuto($Api_Key)).ToString()
+
```

Comment:
Ln:53 of `Get-ITGlue-APIKeyps1` should be changed from `PtrToStringAuto` to `PtrToStringBSTR`

```suggestion
                    ([System.Runtime.InteropServices.Marshal]::PtrToStringBSTR($Api_Key)).ToString()
```

See [PowerShell/PowerShell#12114 (comment)](https://github.com/PowerShell/PowerShell/issues/12114#issuecomment-598192488) for more details.